### PR TITLE
reinsert affinearithmetic.jl

### PR DESCRIPTION
**Pull request by lucaferranti**

### CODE REVIEW

**Review by mforets — APPROVED** [review]

File: src/algorithms.jl [hunk 1]
```diff
@@ -45,6 +45,20 @@ and `f'` is the derivative of `f` (gradient in the multivariate case).
...
+
+"""
+    AffineArithmeticEnclosure <: AbstractDirectRangeAlgorithm
+
+Data type to bound the range of `f` over `X` using affine arithmetic.
+See [`AffineArithmetic.jl`](https://github.com/JuliaIntervals/AffineArithmetic.jl) for more details.
+
+### Notes
+
+To use this algorithm, you need to load `AffineArithmetic.jl`, note also that `AffineArithmetic.jl`
```
Comment (schillic) [inline src/algorithms.jl:57]:
```suggestion
To use this algorithm, you need to load `AffineArithmetic.jl`. Note also that `AffineArithmetic.jl`
```

File: src/algorithms.jl [hunk 2]
```diff
@@ -45,6 +45,20 @@ and `f'` is the derivative of `f` (gradient in the multivariate case).
...
+currently supports only arithmetic operations.
```
Comment (schillic) [inline src/algorithms.jl:58]: As opposed to?

**Review by schillic — APPROVED** [review]

File: src/algorithms.jl (hunk 2 above)
Comment (lucaferranti) [reply src/algorithms.jl:58]: Everything else? I mean AA.jl currently only has `+, -, *, /, ^`

Comment (schillic) [reply src/algorithms.jl:58]: Maybe it is not clear to a reader what else you would want. But I can see that any example would be arbitrary.

Comment (lucaferranti) [reply src/algorithms.jl:58]: Yeah I basically wanted to give a heads up that no elementary function is supported

**Pull request merged**